Bio/SeqFeature.py

```diff
@@ -927,7 +940,21 @@ def nofuzzy_end(self):
             raise
 
     def extract(self, parent_sequence):
-        """Extract feature sequence from the supplied parent sequence."""
+        """Extract the sequence from supplied parent sequence using the FeatureLocation object.
+
+        The parent_sequence can be a Seq like object or a string, and will
+        generally return an object of the same type. The exception to this is
+        a MutableSeq as the parent sequence will return a Seq object.
+
+        >>> from Bio.Seq import Seq
+        >>> from Bio.Alphabet import generic_protein
+        >>> from Bio.SeqFeature import FeatureLocation
+        >>> seq = Seq("MKQHKAMIVALIVICITAVVAAL", generic_protein)
+        >>> fL = FeatureLocation(8, 15)
```

Comment:
That should be ``fl`` (lower case) to match style guidelines and the line below. This is why TravisCI is failing on the doctests.